scripts/ci/format-lintro-pr-comment.py

```
@@ -0,0 +1,465 @@
+            break
+
+    if start_index is None:
+        return None
+
+    summary_lines: list[str] = []
+    for line in lines[start_index : start_index + MAX_SUMMARY_LINES]:
+        if (
+            line.startswith("Cleaning up orphan processes")
+            or line.startswith("[harden-runner]")
+            or "shutdown signal" in line
+            or line.startswith("Post job cleanup")
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)